package.json

```diff
@@ -104,6 +104,7 @@
     "@types/enzyme": "3.10.8",
     "@types/enzyme-adapter-react-16": "1.0.6",
     "@types/jest": "26.0.22",
+    "@types/lodash": "4.14.168",
```

Comment:
do we need this? were is it being used?

Reply:
https://github.com/mattermost/mattermost-mobile/blob/26f6880c047727d7be31f6bf373c1ccb456b51fb/app/components/message_attachments/attachment_footer.js#L8